icons/icq.svg

```diff
@@ -1 +1 @@
.2705 2.0288.2382 3.0079-.2695.9487-.5077 1.5573-1.3222 1.8622-2.3366l.0323-.2018c.6429.7095 1.2515 1.1839 1.9298 1.3525.6762.2038 1.3525.1715 1.962-.2352.5754-.3381.9822-.9145 1.185-1.625.1696-.7449.1696-1.4878 0-2.3336.3382.2372.6763.4038 1.082.5067 1.083.3381 2.0287.3028 2.9775-.2019.9124-.5087 1.522-1.3201 1.8269-2.3688.2705-1.0144.2039-1.9954-.3028-2.9432l-.1716-.2725c-.2695-.4058-.542-.8115-.9488-1.082-.337-.2372-.7418-.4058-1.1143-.5753 1.0497-.5077 1.7926-1.2183 2.1297-2.1317.2382-.7449.1716-1.3545-.1685-1.9631v-.0666c-.4058-.6763-.9458-1.1153-1.7916-1.3525-.5077-.101-1.082-.103-1.5887-.0687l-1.1183.2372.3735-.8458c.337-1.083.2372-2.1317-.3382-3.1127l-.1009-.1716c-.5077-.8458-1.2515-1.3525-2.267-1.622-.2623-.0676-.5207-.114-.775-.1342zm-.5753 1.4272c.326-.0303.656.012.976.1312.7106.1675 1.2183.539 1.524 1.1143v.003c.1697.3381.2696.7428.2696 1.183l-.101.7448c-.1352.3734-.3048.7095-.5076 1.0133l-3.0441 3.7244c-.3028-.2372-.643-.4067-.9811-.5753l-.3391-3.8253-.0333-1.3525.0323-.3381c.1695-.6763.6439-1.184 1.2515-1.522.3048-.1696.6258-.2706.9528-.2998zm-6.9471.542c.1352-.004.2785.0302.4138.0969.6086.1695 1.1163.6086 1.522 1.4534.1716.3391.3392.6762.4068 1.083l.6762 3.9939-.5087.1695-3.0108-3.2823-.1695-.3371c-.2725-.6096-.439-1.184-.3715-1.7936v-.002c.0344-.6086.2705-1.0123.6763-1.2495.101-.0848.2291-.1282.3633-.1312zm12.243 5.0799c.1514 0 .2998.007.4522.0242.5753.0686.981.3401 1.184.7469l.1362.437-.0677.3381c-.2361.6087-.7448 1.084-1.5896 1.4555-.3382.1695-.7096.2372-1.1153.3028l-4.1615.5077-.0676-.2372-.0686-.101.1363-.1685 3.4185-2.8423.4037-.1675.002-.002c.4542-.2019.8933-.2887 1.3404-.2927zm-15.605.1847c.1614.008.327.0333.4945.0757v.002l.9164.3704 2.9775 2.167c-.2725.3391-.439.6763-.541 1.0144l-4.873.6086-.337-.0666c-.6087-.101-1.0164-.3735-1.288-.8781-.2725-.5077-.2694-1.0154-.0666-1.524.2372-.642.6763-1.1507 1.3182-1.4202.4572-.2544.9144-.3765 1.4-.3492zm7.2266 2.715c.6097 0 1.186.2382 1.5584.6097.4744.4743.6763.982.6763 1.5907 0 .6086-.2019 1.1829-.6763 1.5886-.44.44-.9487.6763-1.5573.6763-.642 0-1.181-.2372-1.5877-.6763-.44-.4057-.6762-.979-.6762-1.5886 0-.6087.2361-1.1174.6762-1.5907.4057-.3735.9457-.6097 1.5887-.6097zm8.4935 2.604l.339.0354c.5088.1696.9822.4724 1.2516.979.3392.3735.5077.8812.5077 1.4565l-.1009.6762c-.1686.6763-.5753 1.184-1.1506 1.4535-.6086.339-1.2183.4067-1.8945.1715-.4058-.1009-.6763-.2705-.9124-.541l-3.5528-2.9451.2372-.5067.1009-.4068 3.8233-.3381 1.3525-.0353zm-6.6979 2.4709l1.4201 1.4534.8781 1.083.1716.3402c.3028.5753.4724 1.1153.4038 1.7582-.0667.6076-.2695 1.0477-.6763 1.2172-.2362.1696-.5067.1716-.7772.1363-.6429-.1676-1.1506-.6742-1.522-1.4867l-.4068-1.1184-.5067-2.9442.5733-.2018.4401-.2372zm-3.6537.0686c.3371.1686.6762.3371 1.0497.3704.0666 1.1507.1665 2.369.335 3.6881v1.3545l-.0332.3361c-.2382.6086-.644 1.1173-1.2516 1.3878-.6096.3381-1.2515.4058-1.8622.2372-.6762-.1685-1.184-.5753-1.4534-1.1163-.1716-.3381-.3068-.7469-.3068-1.1506l.1716-.7772c.0666-.3381.2371-.6762.5076-.9488l2.8423-3.3812z"/></svg>
```

Comment:
Looks like this path is at 5 decimals. Any chance you could reduce this?

Reply:
Seems like we have to update the logo anyway: https://www.icq.com
But I couldn't find any SVG file.


As well the old source is down. 😄

Reply:
So I've found a decent sized image comparing the old logo to the new one [here](https://www.underconsideration.com/brandnew/archives/icq_logo_before_after.png).
Been able to extract a path from it, see below:
```svg
M23.382,9.904c-0.516,1.867-2.214,3.124-4.611,3.091c-1.054-0.162-2.539-0.325-3.993-0.632 	c-1.439-0.303-2.077-1.964-1.225-3.172c0.828-1.175,1.714-2.319,2.67-3.391c1.293-1.45,3.431-1.723,5.137-0.768 	C23.068,5.991,23.906,8.009,23.382,9.904z M7.554,14.202c-1.329,0.612-2.641,1.285-3.883,2.056 	c-1.652,1.025-2.267,3.175-1.565,5.025c0.684,1.801,2.578,2.938,4.466,2.681c2.009-0.274,3.5-1.772,3.691-3.746 	c0.073-0.754,0.086-1.514,0.137-2.465c0-0.439,0.021-1.072-0.004-1.703C10.338,14.555,8.899,13.583,7.554,14.202z M17.799,15.423 	c-1.012-0.459-2.042-0.882-3.075-1.291c-1.461-0.579-2.738,0.336-2.677,2.199c0.114,0.913,0.212,2.126,0.425,3.318 	c0.332,1.855,1.899,3.145,3.731,3.164c1.791,0.019,3.386-1.252,3.788-3.017C20.401,17.99,19.522,16.205,17.799,15.423z 	 M13.768,5.075C14.724,2.593,12.899,0.012,9.955,0c-1.14,0.016-2.251,0.637-2.969,1.862C6.251,3.117,6.245,4.433,6.993,5.683 	c0.609,1.018,1.283,1.999,1.98,2.96c0.847,1.168,2.451,1.099,3.15-0.159C12.735,7.382,13.316,6.249,13.768,5.075z M4.212,13.463 	c0.984-0.216,1.957-0.502,2.91-0.83c1.202-0.414,1.5-1.683,0.621-2.608C7.066,9.313,6.363,8.618,5.616,7.98 	C4.624,7.133,3.473,6.977,2.291,7.527C1.112,8.076,0.52,9.055,0.464,10.212C0.446,12.376,2.247,13.894,4.212,13.463z
```

Reply:
I guess we're better off using this version from Wikipedia:

https://en.wikipedia.org/wiki/ICQ#/media/Datei:ICQNewlogo.svg

It matches pretty perfectly with the [PNG in the header of their website](https://icq.com/cached/img/landing/desktop/logo.png).

![image](https://user-images.githubusercontent.com/70264417/103025767-c6ceab80-4552-11eb-9eca-dab3ac232d60.png)